packages/xchain-util/src/asset.test.ts

```diff
@@ -297,6 +302,11 @@ describe('asset', () => {
       const amount = assetAmount(10.01, 8)
       expect(formatAssetAmountCurrency({ amount, asset: AssetBNB, trimZeros: true })).toEqual('10.01 (BNB)')
     })
+
+    it('formats amount by using 0 decimal of asset', () => {
+      const amount = assetAmount(10.01, 0)
+      expect(formatAssetAmountCurrency({ amount, asset: AssetBNB, trimZeros: true })).toEqual('10 (BNB)')
```

Comment:
Same here, pls add a test w/o using trimZeros, but using `decimal` for `formatAssetAmountCurrency`

```
 expect(formatAssetAmountCurrency({ amount, asset: AssetBNB, decimal: 2,  trimZeros: false })).toEqual('10.00 (BNB)')
 ```